src/status_im2/contexts/add_new_contact/events.cljs

```diff
@@ -6,114 +6,133 @@
             [status-im.ethereum.ens :as ens]
             [status-im.ethereum.stateofus :as stateofus]
             [status-im.native-module.core :as status]
-            [status-im2.navigation.events :as navigation]
-            [status-im2.utils.validators :as validators]
+            [status-im2.utils.validators :as v]
             [status-im.utils.utils :as utils]))
 
+;; 1. Domain model
```

Comment:
> Some comments are added to events.cljs for PR reviewers to explain design decisions

@erikseppanen, does that mean you'll remove these comments? They were useful for me as a reviewer, but I guess the code sufficiently explains what it does that these comments are superfluous.

That's a great talk from Sierra btw!

Reply:
Yes, I'll remove them. I should have put them in the PR comments instead. Thanks!

Reply:
done